addons/im_livechat/static/src/core/thread_model_patch.js

```diff
@@ -37,16 +37,20 @@ patch(Thread.prototype, "im_livechat", {
             return `${this.getMemberName(this.correspondent)} (${this.correspondent.country.name})`;
         }
         if (this.channel?.anonymous_country) {
-            return `${this.getMemberName(this.correspondent)} (${this.channel.anonymous_country.name})`;
+            return `${this.getMemberName(this.correspondent)} (${
+                this.channel.anonymous_country.name
+            })`;
         }
         return this.getMemberName(this.correspondent);
     },
 
     get imgUrl() {
-        if (this.type === "livechat" && this.correspondent && !this.correspondent.is_public) {
-            return `/web/image/res.partner/${this.correspondent.id}/avatar_128`;
+        if (this.type !== "livechat") {
+            return this._super();
         }
-        return this._super();
+        return this.correspondent && !this.correspondent.is_public
+            ? `/web/image/res.partner/${this.correspondent.id}/avatar_128`
+            : "/mail/static/src/img/smiley/avatar.jpg";
```

Comment:
Couldn't we simply use avatar Url ? https://github.com/odoo-dev/odoo/blob/master-mail-enable-mention-edit-did/addons/mail/static/src/core/common/thread_service.js#L1044

Reply:
If we do this we should move the `avatarUrl` function to the thread model, move the patches as well and find an alternative for the callers that do not provide the thread parameter. Is it really worth it? @alexkuhn what do you think?

Reply:
I prefer we keep it like this as a quick fix.

We should not move `avatarUrl` to model: `avatarUrl` determines the avatar of the persona, optionally in the context of a `thread`. The models should not have complex logic (dataclasses).

Reply:
At least it should use `DEFAULT_AVATAR` instead if the string.
https://github.com/odoo-dev/odoo/blob/master-mail-display-followers-name-did/addons/mail/static/src/core/common/persona_service.js#L8